pkg/oc/admin/groups/sync/cli/sync.go

```diff
@@ -230,13 +231,28 @@ func (o *SyncOptions) Complete(typeArg, whitelistFile, blacklistFile, configFile
 }
 
 // buildOpenShiftGroupNameList builds a list of OpenShift names from file and args
-func buildOpenShiftGroupNameList(args []string, file string) ([]string, error) {
+// nameMapping is used to override the OpenShift names built from file and args
```

Comment:
bad godoc

Reply:
Er what should I say instead?

Reply:
doesn't match func name

Reply:
It is the second line of the go doc.  The first line correctly states `buildOpenShiftGroupNameList`.

Reply:
oh wow am bad at reading diffs